src/screens/OtherProfile/index.js

```diff
@@ -679,7 +601,7 @@ const OtherProfile = () => {
           onSelect={onBlocking}
           refBlockUser={blockUserRef}
           username={username}
-          isBlocker={blockStatus.blocker}
+          isBlocker={isBlocking}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The prop `isBlocker={isBlocking}` replaces the previous `isBlocker={blockStatus.blocker}`. Ensure that `isBlocking` is equivalent to `blockStatus.blocker` to avoid any behavioral changes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->